lib/Zonemaster/CLI.pm

```diff
@@ -397,20 +406,25 @@ sub run {
                     }
 
                     if ( $self->show_level ) {
-                        printf "%-9s ", translate_severity( $entry->level );
+                        printf "%-9s ", __( $entry->level );
```

Comment:
This breaks msgid extraction. `xgettext` depends on `__()` only ever being called with literal (format) strings.

Reply:
Sorry, I missed the point. It has been fixed.